src/screens/TaskDetailScreen.tsx

```diff
@@ -53,6 +56,39 @@ interface TaskDetailScreenProps {
   navigation: TaskDetailScreenNavigationProp;
 }
 
+// Theme constants
+const COLORS = {
```

Comment:
ou’ve done well keeping styles in StyleSheet.create, but consider extracting all colors into a centralized theme file. That’ll make it easier when you want to support dark mode or tweak the palette.